libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt

```diff
@@ -113,6 +112,7 @@ internal object MapboxNavigationTelemetry :
     private lateinit var locationsCollector: LocationsCollector
     private lateinit var sdkIdentifier: String
     private var logger: Logger? = null
+    private val feedbackEventCacheMap = LinkedHashMap<String, NavigationFeedbackEvent>()
```

Comment:
does the order matters?

Reply:
yes, we want to show the cached feedback events in the order of when it's created.